pkg/test/kube/dump.go

```diff
@@ -440,9 +440,9 @@ func DumpPodProxies(ctx resource.Context, c cluster.Cluster, workDir, namespace
 				return err
 			}
 			defer fw.Close()
+			defer fw.Close()
```

Comment:
Is this meant to be duplicated?

Reply:
nope, that is why the test is failing. will fix. Thanks!